### PR TITLE
OSDOCS-6320 z-stream release notes 4.10.61

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.10

Issue:
https://issues.redhat.com/browse/OSDOCS-6320

Link to docs preview:
[preview](https://file.rdu.redhat.com/cbippley/OSDOCS-6320/release_notes/ocp-4-10-release-notes.html#ocp-4-10-61)

QE review:
- [ ] QE has approved this change.

Additional information:
Z-stream release notes for 4.10.61. [OSDOCS-6320](https://issues.redhat.com/browse/OSDOCS-6320)

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -4039,4 +4039,23 @@ For more information about logging levels for MetalLB, see xref:../networking/me
 [id="ocp-4-10-60-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
```
Comment (dfitzmau) [inline release_notes/ocp-4-10-release-notes.adoc:4042]: This like nitpick goes back a few releases and did not originate with your PR, but we cannot determine when the module name was changed. Perhaps from this release and onwards we can make the following change?

Updating a cluster within a minor version by using the CLI > Updating a cluster using the CLI

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -4039,4 +4039,23 @@ For more information about logging levels for MetalLB, see xref:../networking/me
...
+
+[id="ocp-4-10-61"]
+=== RHSA-2023:3363 - {product-title} 4.10.61 bug fix and security update
+
+Issued: 2023-06-07
+
+{product-title} release 4.10.61 is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:3363[RHSA-2023:3363] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3362[RHSA-2023:3362] advisory.
```
Comment (dfitzmau) [inline release_notes/ocp-4-10-release-notes.adoc:4049]:
```suggestion
{product-title} release 4.10.61, which includes security updates, is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:3363[RHSA-2023:3363] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3362[RHSA-2023:3362] advisory.
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -4039,4 +4039,23 @@ For more information about logging levels for MetalLB, see xref:../networking/me
...
+
+[source,terminal]
+----
+$ oc adm release info 4.10.61 --pullspecs
+----
+
+[id="ocp-4-10-61-upgrading"]
+==== Updating
+
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
```
Comment (dfitzmau) [inline release_notes/ocp-4-10-release-notes.adoc:4061]: Update title as per a previous comment.

**Comment by dfitzmau** [comment]
Nice work, @cbippley . I added comments inline.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (cbippley) [reply release_notes/ocp-4-10-release-notes.adoc:4049]: Done.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 3 above)
Comment (cbippley) [reply release_notes/ocp-4-10-release-notes.adoc:4061]: @dfitzmau Do you it should say:
==== Updating a cluster using the CLI  ?

Comment (dfitzmau) [reply release_notes/ocp-4-10-release-notes.adoc]: Hi @cbippley . The title of this section is OK, but the link anchor text does not match the source heading.

Updating a cluster within a minor version by using the CLI > Updating a cluster using the CLI

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:4042]: You all might want to consider opening a ticket to investigate this xref further and update all of them so the links are consistent throughout the RNs.

**Review by opayne1 — COMMENTED** [review]
Just adding my 2 🪙 in about the xref otherwise lgtm

**Comment by opayne1** [comment]
@dfitzmau I am going to mark this as peer review done so it isnt sitting in the queue.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-10-release-notes.adoc:4042]: I can take this. I'll correct these in future z-stream releases. Thanks, Olivia.

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**